behind-the-veil-siteroot/imports/ui/components/map/MarkerMap.jsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * File Description: A simple map component with a marker at the given location
+ * File version: 1.0
+ * Contributors: Laura
+ */
+
+import React, { useEffect, useState, useRef } from 'react';
+import mapboxgl from 'mapbox-gl';
+import 'mapbox-gl/dist/mapbox-gl.css'
+
+import classNames from "classnames";
+import { mapboxKey, getCoordinates } from './utils'; 
+
+export const MarkerMap = ({ className, location }) => {
+
+    mapboxgl.accessToken = mapboxKey
+
+    const classes = classNames(className, "h-96 w-full");
+
+    const mapContainer = useRef(null);
+    const map = useRef(null);
+    const [zoom, setZoom] = useState(14);
+    const australiaBounds = [[96.8168, -43.7405], [173.0205, -9.1422]];
+
+    useEffect(() => {
+        if (map.current) return;
+
+        const loadMap = async () => {
+            const coordinates = await getCoordinates(location);
+            if (!coordinates) return;
+
+            const mapInstance = new mapboxgl.Map({
+                container: mapContainer.current,
+                style: 'mapbox://styles/mapbox/streets-v11',
+                center: [coordinates.longitude, coordinates.latitude],
+                zoom: zoom,
+                maxBounds: australiaBounds,
+            });
+            map.current = mapInstance;
+
```

Comment:
Documentation also for this file, maybe presenting where the marker is being added in the implementation below